.travis.yml

```diff
@@ -28,7 +28,14 @@ script:
 - yarn test
 - boot test
 - boot release-ci
-- ./build/lumo -c src/test/cljs:src/test/lumo:src/test/cljs_cp scripts/lumo_test.cljs --test-cli-option true
+- boot -qPB with-cp --write --file cp |
```

Comment:
this is not needed

Reply:
I figured we needed a way to resolve the dependencies to `.m2`, so this is one way, we could use `mvn` as well I guess.